public/app/components/panelclipboard.js

```diff
@@ -0,0 +1,31 @@
+define([
+  'lodash',
+  'store'
+],
+function(_, store) {
+  'use strict';
+
+  return {
+    get: function() {
+      var json = store.get("copiedPanel");
+      var panel = null;
+      try {
+        panel = JSON.parse(json);
+      } catch(err) {
+        this.clear();
+      }
+      return panel;
+    },
+    set: function(panel) {
+      var copy = _.clone(panel);
```

Comment:
Angular has a built in copy method already - [seen here](https://docs.angularjs.org/api/ng/function/angular.copy), that is more appropriate and would probably handle the need to manually delete the &&hashkey